apps/crc_interactive.py

```diff
@@ -54,7 +54,7 @@ def __init__(self) -> None:
         resource_args = self.add_argument_group('Arguments for Increased Resources')
         resource_args.add_argument('-b', '--mem', type=int, default=self.default_mem, help='memory in GB')
         resource_args.add_argument(
-            '-t', '--time', default=self.default_time,
+            '-t', '--time', type=self.parse_time, default=self.default_time,
             help=f'run time in hours or hours:minutes [default: {self.default_time} hour] ')
```

Comment:
Minor point, but lets drop the word hour from `[default: {self.default_time} hour]`. I think it reads more nicely with the fstring auto formatting of time objects.

You're welcome to ignore this suggestion if you disagree.